src/resolvers/teams.js

```diff
@@ -24,12 +24,19 @@ const teamsResolvers = {
       });
     },
     async getItemsSold(root, { id }) {
-      return models.Orders.findAll({
-        attributes: ['teamID, numberOfItems', [sequelize.fn('SUM', sequelize.col('numberOfItems')), 'itemsSold']],
+      return models.Team.findOne({
+        attributes: ['itemsSold'],
         where: {
-          teamID: id,
+          teamId: id,
```

Comment:
should `teamID` just be `id`?

Also, thoughts on changing this query to `getSalesInfoForTeam` and returning BOTH the `itemsSold` and `amountRaised`?